src/documents/ComponentGrades.md

```diff
@@ -14,121 +14,87 @@ The Infusion Framework already contains several predefined component grades that
     <thead>
         <tr>
             <th>Grade Name</th>
-            <th>"Relay" Version <sup>[1]</sup></th>
             <th>Description</th>
         </tr>
     </thead>
     <tbody>
         <tr>
-            <td>autoinit</td>
-            <td></td>
+            <td><code>fluid.component</code></td>
             <td>
-                A special directive grade that instructs the framework to automatically construct a globally named creator function (with the same name as the grade) responsible for the construction of the component. NOTE: for the Infusion 2.0 release this grade will become redundant as it will be the default for every grade
-                <br/>
-                <em>
-                    <strong>NOTE:</strong> for the Infusion 2.0 release this grade will become redundant as it will be the default for every grade
-                </em>
+                A "little" component is the most basic component: it supports options merging with defaults (<a href="tutorial-gettingStartedWithInfusion/BasicComponentCreation-LittleComponents.md">Little Components</a>), as well as
+                that instantiating event firers based on default framework events (onCreate, onDestroy, onDetach) 
+                and events declared in the options (<a href="tutorial-gettingStartedWithInfusion/EventedComponents.md">Tutorial - Evented Components</a>. All Fluid components are derived from this grade, and in general all things not derived from this grade are non-components (e.g. plain functions, or model transformation transforms, etc.)
             </td>
         </tr>
         <tr>
-            <td>fluid.littleComponent</td>
-            <td></td>
+            <td><code>fluid.modelComponent</code></td>
             <td>
-                A "little" component is the most basic component: it supports options merging with defaults (<a href="tutorial-gettingStartedWithInfusion/BasicComponentCreation-LittleComponents.md">Little Components</a>). All Fluid components are derived from this grade, and in general all things not derived from this grade are non-components (e.g. plain functions, or model transformation transforms, etc.)
+                A "model" component is a little component that additionally provides supports for a component's model, defined in the components options, and operations on it (<a href="tutorial-gettingStartedWithInfusion/ModelComponents.md">Tutorial - Model Components</a>).
```

Comment:
Is it worth referencing what these "operations" are? i.e. that the component a) has a ChangeApplier automatically assigned to it, and b) supports relays between models? Or is it better to keep it simple and general on this page?